roles/openshift_examples/defaults/main.yml

```diff
@@ -6,7 +6,9 @@ openshift_examples_load_db_templates: true
 openshift_examples_load_xpaas: "{{ openshift_deployment_type in ['enterprise','openshift-enterprise','atomic-enterprise','online']  }}"
 openshift_examples_load_quickstarts: true
 
-examples_base: /usr/share/openshift/examples
+content_version: "{{ 'v1.1' if openshift.common.version_greater_than_3_1_or_1_1 else 'v1.0' }}"
+
+examples_base: "/usr/share/openshift/examples"
```

Comment:
We're not altering the paths on the master, just the content source. Though I do see how that could be confusing. I'm open to landing all versions of content and then installing the appropriate version. If we go that route we'll need to update the docs.